lib/handlers/quote/handler.ts

```diff
@@ -168,6 +174,17 @@ export class QuoteHandler extends APIGLambdaHandler<
     };
   }
 
+  public getQuoteRequestSource(event: APIGatewayProxyEventHeaders): RequestSource {
+    const userAgent = event?.['x-request-source']
+    if (userAgent === undefined) {
+      return RequestSource.UNKNOWN
+    }
+    if (Object.values<string>(RequestSource).includes(userAgent)) {
```

Comment:
hmm I am thinking this might be too restrictive. In other words, if there is a spelling mistake we might just default to a lot of unknowns, and we wont know who is making that mistake. Are we sure we want an enum here?

Reply:
we should at least do some normalization w/ lowercasing for the comparison